mlflow/models/container/__init__.py

```diff
@@ -187,13 +188,22 @@ def _serve_pyfunc(model, env_manager):
         # Since MLServer will run without NGINX, expose the server in the `8080`
         # port, which is the assumed "public" port.
         port = DEFAULT_MLSERVER_PORT
+        model_dict = model.to_dict()
+        inference_server_kwargs = {
```

Comment:
Is this backwards compatible? 
What happens when this model server encounters a model that hasn't been saved with this additional metadata?